CMakeLists.txt

```diff
@@ -40,6 +40,12 @@ ENDIF()
 
 set (WORLD_BUILDER_SOURCE_DIR ${PROJECT_SOURCE_DIR})
 
+# define some config options:
+set(WB_ENABLE_APPS ON CACHE BOOL "If enabled, applications gwb-grid and gwb-dat are built.")
+set(WB_ENABLE_HELPER_TARGETS ON CACHE BOOL "If enabled, targets to indenting and switching to debug/release mode are configured.")
+set(WB_ENABLE_TESTS ON CACHE BOOL "If enabled, tests are configured.")
+set(WB_ENABLE_DOCUMENTATION ON CACHE BOOL "If enabled, building of the documentation is configured.")
```

Comment:
There is already an option called `WB_BUILD_DOCUMENTATION` in `doc/CMakeList.txt`. I am fine with removing that in favor of `WB_ENABLE_DOCUMENTATION`, but that would need to be done.

Reply:
the option I introduced also doesn't do anything. :-) removed.